tests/test_resampler.py

```diff
@@ -128,6 +152,30 @@ def test_resampler_resampling_function_last() -> None:
     assert resampled == expected
 
 
+def test_resampler_resampling_function_coalesce() -> None:
+    """Test the resampler."""
+    start = dt.datetime(1970, 1, 1, tzinfo=dt.timezone.utc)
+    step = dt.timedelta(seconds=1)
+    resampler = Resampler(
+        dt.timedelta(seconds=5),
+        ResamplingFunction.Coalesce,
+        max_age_in_intervals=1,
+        start=start,
+    )
+
+    for i in range(10):
+        resampler.push_sample(timestamp=start + i * step, value=i + 1)
+
+    expected = [
+        (start + 5 * step, 1.0),
+        (start + 10 * step, 6.0),
```

Comment:
I suggest to have some Nones in the test case

Reply:
There are already some in the Rust tests, but I will add some here as well.